.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 npm-debug.log
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+.env
```

Comment:
metadata LGTM, but can you move it to a separate PR? we can probably merge it immediately